### PR TITLE
Pulsing Dots with less

### LINKED ISSUE #3097: Improve pulsing dots performance
### Expected Behavior

Use animated interpolation

### Current Behavior

Updates state

### PR DESCRIPTION
**Pull request by aaronmgdr**
## Description

Update Pulsing Dots To Only use one animated value and use interpolation for the rest. 

also remove an event listener that was being called on each render.

## Other changes

made some additions to getting started on ios readme

## Tested

ran simulator and saw the dots pulse
## Related issues

- Fixes #3097 

## Commit Message
- Fixes #3097 pulsing dots performance

### CODE REVIEW

**Comment by aaronmgdr** [comment]
JK doing more work

File: SETUP.md [hunk 1]
```diff
@@ -121,11 +121,16 @@ sdkmanager 'platforms;android-28'
...
 Install [Xcode 10.3](https://download.developer.apple.com/Developer_Tools/Xcode_10.3/Xcode_10.3.xip) (an Apple Developer Account is needed to access this link).
+(If after signing in the the direct link does not work try finding on https://developer.apple.com/download/more/)
 
 We do not recommend installing Xcode through the App Store as it can auto update and become incompatible with our projects (until we decide to upgrade).
 
 Note that using the method above, you can have multiple versions of Xcode installed in parallel by using different app names. For instance `Xcode10.3.app` and `Xcode11.app` inside the `/Applications` folder.
 
+Install Cocopods `gem install cocoapods`
+
+From ios folder in mobile package run `bundle exec pod install`
```
Comment (jmrossy) [inline SETUP.md:132]: 🙏

**Review by jmrossy — APPROVED** [review]
Nice!!

**Comment by aaronmgdr** [comment]
Confirmed on android phone

**Pull request merged**